src/store/metadata/stub/content/editions/local-authority-greenhouse-gas-emissions_2023-06.json

```diff
@@ -30,76 +31,77 @@
         "country"
       ],
       "licence": "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/",
-      "issued": "2023-06-29T13:30:00Z",
-      "modified": "2023-06-29T16:20:00Z",
+      "issued": "2023-06-29T00:00:00Z",
+      "modified": "2023-10-31T00:00:00Z",
       "spatial_resolution": [
         "E01",
         "E92"
       ],
       "spatial_coverage": "E92000001",
       "temporal_resolution": [
         "P1M",
-        "P3M"
+        "P3M",
+        "P1Y"
       ],
       "temporal_coverage": {
         "start": "2005-01-01T00:00:00Z",
         "end": "2022-07-01T00:00:00Z"
       },
-      "versions_url": "https://staging.idpd.uk/datasets/beis-local-authority-greenhouse-gas-emissions/editions/2022-07/versions",
+      "versions_url": "https://staging.idpd.uk/datasets/local-authority-greenhouse-gas-emissions/editions/2023-06/versions",
       "versions": [
         {
-          "@id": "https://staging.idpd.uk/datasets/beis-local-authority-greenhouse-gas-emissions/editions/2022-07/versions/1",
+          "@id": "https://staging.idpd.uk/datasets/local-authority-greenhouse-gas-emissions/editions/2023-06/versions/1",
           "issued": "2023-06-29T00:00:00Z"
         }
       ],
-      "next_release": "2023-07-06T09:30:00Z",
+      "next_release": "2024-06-06T09:30:00Z",
       "table_schema": {
         "columns": [
           {
             "name": "year",
-            "datatype": "xsd:datetime",
+            "datatype": "datetime",
```

Comment:
Year is using a datetime datatype. Either the datatype or the name of the column needs to be changed.